sdk/loadtestservice/azure-developer-loadtesting/azure/developer/loadtesting/_patch.py

```diff
@@ -6,69 +6,42 @@
-
-        AppComponentOperations.__init__(self, self._client, self._config, self._serialize, self._deserialize)
-        ServerMetricsOperations.__init__(self, self._client, self._config, self._serialize, self._deserialize)
-        TestOperations.__init__(self, self._client, self._config, self._serialize, self._deserialize)
-
-
-class LoadTestingClient(LoadTestingClientGenerated):  # pylint: disable=client-accepts-api-version-keyword
-    """These APIs allow end users to create, view and run load tests using Azure Load Test Service.
-
-    :ivar app_component: AppComponentOperations operations
-    :vartype app_component: azure.developer.loadtesting.operations.AppComponentOperations
-    :ivar server_metrics: ServerMetricsOperations operations
-    :vartype server_metrics: azure.developer.loadtesting.operations.ServerMetricsOperations
-    :ivar test: TestOperations operations
-    :vartype test: azure.developer.loadtesting.operations.TestOperations
-    :ivar test_run: TestRunOperations operations
-    :vartype test_run: azure.developer.loadtesting.operations.TestRunOperations
-    :param endpoint: URL to perform data plane API operations on the resource. Required.
-    :type endpoint: str
-    :param credential: Credential needed for the client to connect to Azure. Required.
-    :type credential: ~azure.core.credentials.TokenCredential
-    :keyword api_version: Api Version. Default value is "2022-06-01-preview". Note that overriding
-     this default value may result in unsupported behavior.
-    :paramtype api_version: str
-    """
-
-    def __init__(self, endpoint: str, credential: "TokenCredential", **kwargs: Any) -> None:
-        self._config = LoadTestingClientConfiguration(endpoint=endpoint, credential=credential, **kwargs)
-        self._client = PipelineClient(base_url=endpoint, config=self._config, **kwargs)
-
-        self._serialize = Serializer()
-        self._deserialize = Deserializer()
-        self._serialize.client_side_validation = False
-
-        self.load_test_runs = TestRunOperations(self._client, self._config, self._serialize, self._deserialize)
-        self.load_test_administration = LoadTestingAdministration(
-            self._client, self._config, self._serialize, self._deserialize
-        )
-
-
-__all__: List[str] = ["LoadTestingClient"]  # Add all objects you want publicly available to users at this package level
+from typing import List
+from enum import Enum
+
+
+class TestFileValidationStatus(Enum):
+    ValidationInitiated = 0
+    ValidationSuccess = 1
+    ValidationFailed = 2
+    ValidationCheckTimeout = 3
+
+
+class TestRunStatus(Enum):
+    Accepted = 0
+    NotStarted = 1
+    Provisioning = 2
+    Provisioned = 3
+    Configuring = 4
+    Configured = 5
+    Executing = 6
+    Executed = 7
+    Deprovisioning = 8
+    Deprovisioned = 9
+    Done = 10
+    Cancelling = 11
+    Cancelled = 12
+    Failed = 13
+    ValidationSuccess = 14
+    ValidationFailed = 15
+    CheckTimeout = 16
```

Comment:
(1) Why are there some empty lines?
(2) Why don't you define it in swagger if they are used by some operations?

Reply:
I am not sure, if I am able to understand you completely, can you please help me with it?

Reply:
If you can define these enum value in swagger, autorest can generate it in `_enum.py` so that you don't need to write it manually. Could you provide context about these enum value?

Reply:
Hi @msyyc, 
There are 2 enums we are using.

First, TestFileValidationStatus, this is not in swagger its corresponding function is not also in swagger, this is for our customer wrapper/LRO which we are implementing, to check the validation status of loadtest script file (.JMX)

Second, this is in our swagger, not sure why it not being generated. 
![image](https://user-images.githubusercontent.com/40313233/202966744-7ef5680b-c1ce-4ef3-a57f-d74effe86808.png)

Reply:
I believe it is related with your directives in README.md. Could you check it?